### ISSUE TITLE
Require tickets to have unique phone?

### ISSUE THREAD

**Issue by ryandiks** [natedl98/csr-rh#36]
How does this system work? Does it?

From the "Edit Campaign" screen:

![image](https://user-images.githubusercontent.com/49453047/63122872-5763a780-bf6d-11e9-83ed-0a9e01f709ec.png)

**Comment by natedl98** [comment]
I don't think this works 100% with the new system (although it did with the old system). It'll get taken care of after some of these other things, unless it's a priority?
